app/TracingStrategyAssets.js

```diff
@@ -0,0 +1,80 @@
+import { Images } from './assets';
+import { config } from './COVIDSafePathsConfig';
+import languages from './locales/languages';
+
+export const isGPS = config.tracingStrategy === 'gps';
+
```

Comment:
I see this file as potentially being a pain to maintain over time.  Do you have a sense of exactly how much of the copy will be different between the two apps at this point?  if its about this much then i wouldnt worry about it, but if its going to be a lot i might consider trying to get something more programatic working.

do you think something like this would be viable?

```javascript
const myTranslate = (translation) => {
  const translation = isGPS ? languages.t(`${translation}_location`) : languages.t(`${translation}_bluetooth`)
  return translation
}
```

I think your idea from before was really good as well and would be ideal if we could get it to work. I think the issue we had may have been resolved with the switch from passing the tracingStrategy from over the bridge to having it be a part of the preprocess step via react-native-config.

that is, something like: 

```javascript
const myTranslate = (translation) => {
  const translation = isGPS ? languages.t(translation).gps : languages.t(translation).bte
  return translation
}
```

or if we could get an api like this to work that would be even better:

```javascript
  <Text>{t('label.foo.bar')[tracingStrategy]}</Text>
```

Didn't actually try it myself, so Ill defer to you if this is a good idea or not, but it would be nice to not need to maintain an additional assets file.

Reply:
Please do not do string manipulation or use variables inside `t(x)` otherwise the `yarn i18n:extract` will not find things. This tool helps keep us sane with so many strings around.

Reply:
good to know, I take back all my suggestions!